### PR TITLE
tiflash: add more explanation on how to clear syncing rules about tiflash in pd

### PR DESCRIPTION
**Pull request by lidezhu**
### What is changed, added or deleted? (Required)
add more explanation on how to clear syncing rules about tiflash in pd.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: reference/tiflash/maintain.md [hunk 1]
```diff
@@ -58,8 +58,54 @@ category: reference
...
->
-> 手动在 PD 中清除同步规则的方法是：发送 DELETE 请求 `http://<pd_ip>:<pd_port>/pd/api/v1/config/rule/tiflash/<rule_id>`，rule_id 是需要清除的 rule 的 id。
+
+ 手动在 PD 中清除同步规则的方法如下：
+1. 查询当前 PD 中所有的数据同步规则，其中 `group_id` 为 `tiflash` 的规则均是与 TiFlash 相关的数据同步规则。
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    curl http://<pd_ip>:<pd_port>/pd/api/v1/config/rules
```
Comment (solotzg) [inline reference/tiflash/maintain.md:68]: curl http://<pd_ip>:<pd_port>/pd/api/v1/config/rules/group/tiflash

Comment (lidezhu) [reply reference/tiflash/maintain.md]: Fixed.

File: reference/tiflash/maintain.md [hunk 2]
```diff
@@ -58,8 +58,46 @@ category: reference
 > **注意：**
 >
 > 如果在集群中所有的 TiFlash 节点停止运行之前，没有取消所有同步到 TiFlash 的表，则需要手动在 PD 中清除同步规则，否则无法成功完成 TiFlash 节点的下线。
->
-> 手动在 PD 中清除同步规则的方法是：发送 DELETE 请求 `http://<pd_ip>:<pd_port>/pd/api/v1/config/rule/tiflash/<rule_id>`，rule_id 是需要清除的 rule 的 id。
+
+ 手动在 PD 中清除同步规则的方法如下：
```
Comment (yikeke) [inline reference/tiflash/maintain.md:62]:
```suggestion
手动在 PD 中清除同步规则的方法如下：

```

File: reference/tiflash/maintain.md [hunk 3]
```diff
@@ -58,8 +58,46 @@ category: reference
...
+1. 查询当前 PD 中所有与 TiFlash 相关的的数据同步规则。
```
Comment (yikeke) [inline reference/tiflash/maintain.md:63]:
```suggestion
1. 查询当前 PD 实例中所有与 TiFlash 相关的的数据同步规则。
```

File: reference/tiflash/maintain.md [hunk 4]
```diff
@@ -58,8 +58,47 @@ category: reference
 > **注意：**
 >
 > 如果在集群中所有的 TiFlash 节点停止运行之前，没有取消所有同步到 TiFlash 的表，则需要手动在 PD 中清除同步规则，否则无法成功完成 TiFlash 节点的下线。
->
-> 手动在 PD 中清除同步规则的方法是：发送 DELETE 请求 `http://<pd_ip>:<pd_port>/pd/api/v1/config/rule/tiflash/<rule_id>`，rule_id 是需要清除的 rule 的 id。
+
+手动在 PD 中清除同步规则的方法如下：
```
Comment (yikeke) [inline reference/tiflash/maintain.md:62]:
```suggestion
手动在 PD 中清除同步规则的步骤如下：
```

File: reference/tiflash/maintain.md [hunk 5]
```diff
@@ -58,8 +58,47 @@ category: reference
 > **注意：**
 >
 > 如果在集群中所有的 TiFlash 节点停止运行之前，没有取消所有同步到 TiFlash 的表，则需要手动在 PD 中清除同步规则，否则无法成功完成 TiFlash 节点的下线。
->
-> 手动在 PD 中清除同步规则的方法是：发送 DELETE 请求 `http://<pd_ip>:<pd_port>/pd/api/v1/config/rule/tiflash/<rule_id>`，rule_id 是需要清除的 rule 的 id。
+
+手动在 PD <span class="x x-first x-last">中清除同步规则的步骤如下</span>：
```
Comment (yikeke) [inline reference/tiflash/maintain.md:62]:
```suggestion
手动在 PD 中清除同步规则的步骤如下：
```

File: reference/tiflash/maintain.md [hunk 6]
```diff
@@ -58,8 +58,47 @@ category: reference
 > **注意：**
 >
 > 如果在集群中所有的 TiFlash 节点停止运行之前，没有取消所有同步到 TiFlash 的表，则需要手动在 PD 中清除同步规则，否则无法成功完成 TiFlash 节点的下线。
->
-> 手动在 PD 中清除同步规则的方法是：发送 DELETE 请求 `http://<pd_ip>:<pd_port>/pd/api/v1/config/rule/tiflash/<rule_id>`，rule_id 是需要清除的 rule 的 id。
+
+手动在 PD 中清除同步规则的步骤如下：
+
+1. 查询当前 PD 实例中所有与 TiFlash 相关的的数据同步规则。
```
Comment (ran-huang) [inline reference/tiflash/maintain.md:64]:
```suggestion
1. 查询当前 PD 实例中所有与 TiFlash 相关的数据同步规则。
```

Comment (yikeke) [reply reference/tiflash/maintain.md:64]: Can you open a PR to fix this typo? @lidezhu

**Pull request merged**